.github/workflows/ci.yml

```diff
@@ -4,10 +4,13 @@ jobs:
   test:
     name: Lint & Test
     runs-on: ubuntu-16.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - uses: actions/checkout@v1
       - uses: bahmutov/npm-install@v1
       - run: yarn ci && node_modules/.bin/codecov
+      - uses: danger/danger-js@9.1.8
```

Comment:
I'd recommend against the GitHub action and use yarn, maybe I should just outright remove the action - I think it's a bad abstraction and probably the reason why you have weird errors

Reply:
good to know! good idea too, yeah. an action really doesn't even make sense at this, I hadn't thought about collecting test coverage when it started. I'll just run it after codecov. thanks!